src/main/java/io/stargate/sgv2/jsonapi/exception/ErrorCodeV1.java

```diff
@@ -230,9 +225,6 @@ public JsonApiException toApiException(Throwable cause, String format, Object...
   }
 
   private String getErrorMessage(String format, Object... args) {
-    if (ExtendError.enabled()) {
```

Comment:
This is the real fix: V2 handling should not be imposed on V1 due to incompatible handling -- change to V2 should be done by upgrading to new `ErrorCode` approach, on per error-code instance (moving over all `ErrorCodeV1` uses one by one).